src/routes/Login/index.tsx

```diff
@@ -1,12 +1,60 @@
+import { ChangeEvent, FormEvent, useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import store from 'store'
+
+import { checkValidation } from 'services/user'
+
 import styles from './login.module.scss'
-import { useState } from 'react'
 import loginBackgroundImg from '../../assets/images/loginBackgroundImg.jpg'
 import { FaEye, FaEyeSlash, FaUser, FaTrash } from 'react-icons/fa'
 
 const UserManage = () => {
-  const [showPassword, setShowPassword] = useState(false)
+  const [adminId, setAdminId] = useState('')
   const [password, setPassword] = useState('')
-  const [userId, setUserId] = useState('')
+
+  const [showPassword, setShowPassword] = useState(false)
+  const [errorMessage, setErrorMessage] = useState(false)
+  const [checked, setChecked] = useState(false)
+
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    if (sessionStorage.getItem('admin') || store.get('admin')) {
+      navigate('/')
+    }
+  }, [navigate])
+
+  const handleIdInput = (e: ChangeEvent<HTMLInputElement>) => {
+    setAdminId(e.currentTarget.value)
+    setErrorMessage(false)
+  }
+
+  const handlePasswordInput = (e: ChangeEvent<HTMLInputElement>) => {
+    setPassword(e.currentTarget.value)
+    setErrorMessage(false)
+  }
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    if (checkValidation(adminId, password) && checked) {
+      sessionStorage.setItem('admin', adminId)
+      store.set('admin', adminId)
+      navigate('/')
+    } else if (checkValidation(adminId, password)) {
+      sessionStorage.setItem('admin', adminId)
+      navigate('/')
```

Comment:
```suggestion
    if (checkValidation(adminId, password) && checked) {
      ichecked ? store.set('admin', adminId) : store.remove('admin', adminId) // 아마 store에 있는지 확인 안하면 remove는 에러 날지도?
      sessionStorage.setItem('admin', adminId)
      navigate('/')
```
checkValidation(adminId, password) 이 부분에 대한 로직이 반복되니까 이렇게 하는건 어떨까요

Reply:
위에 두개도 bool형인데요? 거기도 is 붙이셔야 됩니당